Automattic/pocket-casts-android
# Fix OPML import failing with invalid XML characters.

This change fixes the issue that many users can't import their OPML files because the XML is invalid. If the SAX parser fails it tries searching for the feed URLs using a regular expression.

It also upgrades the Mockito libraries. Part of this includes the move of the com.nhaarman.mockitokotlin2 library to the new package org.mockito.kotlin.

**Test Steps**
1. Load the test OPML onto your device
2. Tap the Profile tab
3. Tap the settings cog
4. Tap "Import & export OPML"
5. Tap "Select file"
6. Choose the file 

The two podcasts should be added to your collection.

[test.opml.zip](https://github.com/Automattic/pocket-casts-android/files/9315851/test.opml.zip)

Fixes https://github.com/Automattic/pocket-casts-android/issues/51
